### PR TITLE
Add `WP_VERSION` env variable to allow specific WP versions.

### PR DESCRIPTION
**Pull request by dsifford**
- Bump version number to 0.11.0
- Update CHANGELOG
- Update README.md

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -225,8 +226,7 @@ check_themes() {
 
     theme_url=${theme_url:-$theme_name}
 
-    WP theme is-installed "$theme_name"
-    if [ $? -eq 0 ]; then
```
Comment (dsifford) [inline run.sh]: This got flagged by shellcheck as a redundancy (in two place). That's why it was changed.

File: run.sh [hunk 2]
```diff
@@ -56,7 +57,7 @@ main() {
     h2 "Installing WordPress"
     h3 "Downloading..."
     chown -R www-data:www-data /app /var/www/html
-    WP core download |& loglevel
+    WP core download --version="$WP_VERSION" |& loglevel
```
Comment (dsifford) [inline run.sh]: Added this so I can test `nightly` builds as needed.

File: README.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ services:
       PLUGINS: >-
         academic-bloggers-toolkit,
         co-authors-plus,
-        [WP-API-master]https://github.com/WP-API/WP-API/archive/master.zip,
+        [WP-API]https://github.com/WP-API/WP-API/archive/master.zip,
```
Comment (dsifford) [inline README.md]: The need for `-master` was revoked a few versions ago with WP-CLI. They now correct for this internally.

**Comment by karellm** [comment]
@dsifford All good, thanks!

**Pull request merged**